package.json

```diff
@@ -69,6 +69,7 @@
     "plotly.js": "1.30.1",
     "prop-types": "^15.6.1",
     "react": "^16.3.2",
+    "react-ace": "^6.1.0",
```

Comment:
We should probably remove `brace` from the dependencies?

Reply:
done

Reply:
oops, we do import `brace` and use it directly, so leaving it in dependencies.

Reply:
I don't think you need to declare it here to require from it, as it's a dependency of react-ace. Did you encounter an issue after removing it from `package.json`?

Reply:
Removing it gives linter errors (and I'm reluctant in general to not explicitly list all dependencies).